docs/source/process.rst

```diff
@@ -0,0 +1,570 @@
+   >>>
+   >>> bookcorpus = load_dataset("bookcorpus", split="train")
+   >>> wiki = load_dataset("wikipedia", "20200501.en", split="train")
+   >>> wiki = wiki.remove_columns("title")  # only keep the text
+   >>>
+   >>> assert bookcorpus.features.type == wiki.features.type
+   >>> bert_dataset = concatenate_datasets([bookcorpus, wiki])
+
+.. seealso::
+
+    You can also mix several datasets together by taking alternating examples from each one to create a new dataset. This is known as interleaving datasets, and you can use it with :func:`datasets.interleave_datasets`.  See the ``IterableDataset`` section for an example of how it's used.
+
+``map``
+-------
+
+Some of the more powerful applications provided by Datasets come from using :func:`datasets.Dataset.map`. It allows you to apply a processing function to each example in a dataset, independently or in batches. This function can even create new rows and columns. 
+
+The primary utility of :func:`datasets.Dataset.map` is to update and modify the contents of a dataset. In the following example, you will prefix each ``sentence1`` value in the dataset with ``'My sentence: '``. First, create a function that will add ``'My sentence: '`` to the beginning of each sentence. The function needs to accept and outputs a :obj:`dict`:
+
+    >>> def add_prefix(example):
+    ...     example['sentence1'] = 'My sentence: ' + example['sentence1']
+    ...     return example
+        
+Next, apply this function to your dataset with :func:`datasets.Dataset.map`:
+
+    >>> updated_dataset = small_dataset.map(add_prefix)
+    >>> updated_dataset['sentence1'][:5]
+    ['My sentence: Amrozi accused his brother , whom he called " the witness " , of deliberately distorting his evidence .',
+    "My sentence: Yucaipa owned Dominick 's before selling the chain to Safeway in 1998 for $ 2.5 billion .",
+    'My sentence: They had published an advertisement on the Internet on June 10 , offering the cargo for sale , he added .',
+    'My sentence: Around 0335 GMT , Tab shares were up 19 cents , or 4.4 % , at A $ 4.56 , having earlier set a record high of A $ 4.57 .',
+    ]
+
+Let's take a look at another example, except this time, you will remove a column with :func:`datasets.Dataset.map`. When you remove a column, it is only removed after the example has been provided to the mapped function. This allows the mapped function to use the content of the columns before they are removed. 
+
+Specify the column to remove with the ``remove_columns=List[str]`` argument in :func:`datasets.Dataset.map`:
+
+    >>> updated_dataset = dataset.map(lambda example: {'new_sentence': example['sentence1']}, remove_columns=['sentence1'])
+    >>> updated_dataset.column_names
+    ['sentence2', 'label', 'idx', 'new_sentence']
+
+.. tip::
+
+    Datasets also has a :func:`datasets.Dataset.remove_columns` method that is functionally identical, but faster, because it doesn't copy the data to a new dataset object.
```

Comment:
```suggestion
    Datasets also has a :func:`datasets.Dataset.remove_columns` method that is functionally identical, but faster, because it doesn't copy the data of the remaining columns.
```